lib/iso_builder.py

```diff
@@ -155,18 +155,12 @@ def _save(self):
                                   LATEST_SYMLINK_NAME)
         utils.force_symlink(self.timestamp, latest_dir)
 
-        iso_file = "%s-DVD-%s-%s.iso" % (self.distro, self.arch, self.version)
-        checksum_file = ("%s-%s-%s-CHECKSUM" %
-                         (self.distro, self.version, self.arch))
         iso_dir = "/%s/%s/iso" % (self.version, self.arch)
-        iso_path = os.path.join(iso_dir, iso_file)
-        checksum_path = os.path.join(iso_dir, checksum_file)
-        chroot_files = "%s %s" % (iso_path, checksum_path)
+        iso_files = "%s/*" % iso_dir
```

Comment:
I'd call this `iso_files_pattern` so no one confuses it with a list of files, but that's a matter of taste.